### PR TITLE
added the terms and policy pages

### LINKED ISSUE #206: Develop Term of Use and Policy pages
## Summary

- Develop the Terms & Condition page
- Develop the Privacy Policy Page

### PR DESCRIPTION
**Pull request by ScottDeev**
## 🛠️ Fixes Issue

Closes #206 

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
Screenshot (254)
Screenshot (255)
Screenshot (256)

### CODE REVIEW

**Review by t-bello7 — DISMISSED** [review]
Good to merge.

File: styles/index.css [hunk 1]
```diff
@@ -5,7 +5,9 @@
 @tailwind utilities;
 @tailwind utilities;
 
-
+.active{
+    color: #0F27BD;
+}
```
Comment (dprincecoder) [inline styles/index.css:10]: I don't think this is necessary here, you can add it to the file you made changes to, so this won't affect other people's work with the same class.

**Review by dprincecoder — CHANGES_REQUESTED** [review]

File: styles/index.css (hunk 1 above)
Comment (ScottDeev) [reply styles/index.css:10]: Alright. I will do just that

**Comment by ScottDeev** [comment]
@dprincecoder I created a unique css file for that style. That way, the style won't affect other files it's not imported into

**Review by dprincecoder — DISMISSED** [review]

**Comment by dprincecoder** [comment]
Good to merge

**Comment by ingressive4good** [comment]
there is a conflict

**Comment by ScottDeev** [comment]
fixed @ingressive4good

**Comment by dprincecoder** [comment]
@ingressive4good good to merge

**Pull request merged**